{"name": "测试数据：【 135，123456，13500010014，1，深圳市宝安区，True，0，修改用户信息成功】", "status": "broken", "statusDetails": {"message": "AttributeError: 'Response' object has no attribute 'response'", "trace": "@pytest.fixture(scope=\"session\")\n    def login_fixture():\n        username = base_data[\"init_admin_user\"][\"username\"]\n        password = base_data[\"init_admin_user\"][\"password\"]\n        header = {\n            \"Content-Type\": \"application/x-www-form-urlencoded\"\n        }\n        payload = {\n            \"username\": username,\n            \"password\": password\n        }\n        loginInfo = user.login(data=payload, headers=header)\n        step_login(username, password)\n        # yield loginInfo.json()\n>       yield loginInfo.response.json()\nE       AttributeError: 'Response' object has no attribute 'response'\n\ntestcases/conftest.py:57: AttributeError"}, "description": "该用例是针对获取用户修改接口的测试", "attachments": [{"name": "log", "source": "3ad50084-c446-471b-9aa9-d54227637d47-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "895ce33b-49fb-47f4-84d6-bd1205b30bab-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "id", "value": "135"}, {"name": "new_password", "value": "'123456'"}, {"name": "new_telephone", "value": "'13500010014'"}, {"name": "new_sex", "value": "'1'"}, {"name": "new_address", "value": "'深圳市宝安区'"}, {"name": "except_result", "value": "True"}, {"name": "except_code", "value": "0"}, {"name": "except_msg", "value": "'修改用户信息成功'"}], "start": 1744986708393, "stop": 1744986708393, "uuid": "a7b089cd-e67d-4b2a-a3da-14b01bc783bc", "historyId": "98cc19cc2b30498d5a3efe57aabfcbeb", "testCaseId": "6ac2471905e3fb58977e6583fe072400", "fullName": "testcases.api_test.test_04_update_user.TestUpdate#test_update_user", "labels": [{"name": "story", "value": "用例--修改用户信息"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "feature", "value": "用户修改模块"}, {"name": "severity", "value": "normal"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_04_update_user"}, {"name": "subSuite", "value": "TestUpdate"}, {"name": "host", "value": "192.168.10.5"}, {"name": "thread", "value": "42260-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_04_update_user"}], "links": [{"type": "test_case", "url": "https://blog.csdn.net/", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://blog.csdn.net/", "name": "点击，跳转到对应BUG的链接地址"}]}